build_ngtf.py

```diff
@@ -334,104 +308,54 @@ def main():
-        # Build nGraph if required.
-        if not arguments.use_prebuilt_ngraph:
-            ngraph_src_dir = './ngraph'
-            if arguments.ngraph_src_dir:
-                ngraph_src_dir = arguments.ngraph_src_dir
-                print("Using local nGraph source in directory ", ngraph_src_dir)
-            else:
-                if arguments.ngraph_version:
-                    ngraph_version = arguments.ngraph_version
-
-                print("nGraph Version: ", ngraph_version)
-                download_repo("ngraph",
-                              "https://github.com/NervanaSystems/ngraph.git",
-                              ngraph_version)
-
-            # Now build nGraph
-            ngraph_cmake_flags = [
-                "-DNGRAPH_INSTALL_PREFIX=" + artifacts_location,
-                "-DNGRAPH_USE_CXX_ABI=" + cxx_abi, "-DNGRAPH_DEX_ONLY=TRUE",
-                "-DNGRAPH_DEBUG_ENABLE=NO", "-DNGRAPH_UNIT_TEST_ENABLE=NO",
-                "-DNGRAPH_TARGET_ARCH=" + target_arch,
-                "-DNGRAPH_TUNE_ARCH=" + target_arch, "-DNGRAPH_TBB_ENABLE=FALSE"
-            ]
-
-            if arguments.debug_build:
-                ngraph_cmake_flags.extend(["-DCMAKE_BUILD_TYPE=Debug"])
-
-            ngraph_cmake_flags.extend([
-                "-DNGRAPH_TOOLS_ENABLE=" +
-                flag_string_map[platform.system() != 'Darwin']
-            ])
-
-            cmake_build(build_dir, ngraph_src_dir, ngraph_cmake_flags,
-                        verbosity)
+    if not arguments.use_prebuilt_openvino:
+        openvino_version = "releases/2021/1"
+        openvino_src_dir = "./openvino"
+        download_repo(
+            "openvino",
+            "https://github.com/openvinotoolkit/openvino",
+            openvino_version,
+            submodule_update=True)
+
+        # Now build OpenVINO
+        openvino_cmake_flags = [
+            "-DENABLE_V7_SERIALIZE=ON",
+            "-DENABLE_TESTS=OFF",
+            "-DENABLE_SAMPLES=OFF",
+            "-DENABLE_FUNCTIONAL_TESTS=OFF",
+            "-DENABLE_VPU=OFF",  # TODO: Fix OpenVINO VPU build
+            "-DNGRAPH_USE_CXX_ABI=" + cxx_abi,
+            "-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=" + cxx_abi,
+            "-DENABLE_CPPLINT=OFF",
+            "-DENABLE_SPEECH_DEMO=FALSE",
+            "-DCMAKE_INSTALL_RPATH=\"$ORIGIN\"",
+            "-DCMAKE_INSTALL_PREFIX=" + os.path.join(artifacts_location,
+                                                     "openvino")
+        ]
+
+        if arguments.debug_build:
+            openvino_cmake_flags.extend(["-DCMAKE_BUILD_TYPE=Debug"])
+
+        cmake_build(build_dir, openvino_src_dir, openvino_cmake_flags,
+                    verbosity)
 
     # Next build CMAKE options for the bridge
     ngraph_tf_cmake_flags = [
         "-DNGRAPH_TF_INSTALL_PREFIX=" + artifacts_location,
-        "-DUSE_PRE_BUILT_NGRAPH=ON",
         "-DNGRAPH_TARGET_ARCH=" + target_arch,
         "-DNGRAPH_TUNE_ARCH=" + target_arch,
```

Comment:
Are these flags required for the bridge?

Reply:
No they're not. Good catch, I'll remove them from the build script too.

Reply:
Actually, I added it to the bridge build just in case, but most importantly also pass it to OV's build (d0d573494)